### PR TITLE
Add simulation tests to `lrauv_system_tests` package.

### PR DESCRIPTION
**Pull request by hidmic**
Depends on https://github.com/osrf/lrauv/pull/195. For review, focus on https://github.com/osrf/lrauv/commit/572a9de25bbbc686cc09898af410894882ad8385 until rebase.

### CODE REVIEW

**Comment by arjo129** [comment]
I've re-based the PR to #195 for easier review.  Github should automatically rebase it to main when we merge #195.

File: lrauv_system_tests/CMakeLists.txt [hunk 1]
```diff
@@ -44,3 +44,11 @@ target_link_libraries(${PROJECT_NAME}_support PUBLIC
   lrauv_ignition_plugins::lrauv_state
   gtest)
 target_compile_features(${PROJECT_NAME}_support PUBLIC cxx_std_17)
+#============================================================================
+# Tests
+if(BUILD_TESTING)
+  enable_testing()
+  include(Dart)
```
Comment (arjo129) [inline lrauv_system_tests/CMakeLists.txt:51]: Are we calling DART directly anywhere in our tests? If not, can this be removed.

**Review by arjo129 — APPROVED** [review]
I ran these locally and they pass. I also checked and they seem to do the same thing as the previous tests.

File: lrauv_system_tests/CMakeLists.txt (hunk 1 above)
Comment (hidmic) [reply lrauv_system_tests/CMakeLists.txt]: Amended in 937013dacc73ba4c2581150af8585b480f713006.

**Pull request merged**